int32 JOINING = 0
int32 LEAVING = 1
int32 NORMAL = 2

uint32 seq
int32 person_id
int32 group_id
int32[] people_in_group
string message_type